### ISSUE TITLE
Custom path not usable for Dashboard

### ISSUE THREAD

**Issue by dpvDuncan** [containous/traefik#3282]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I wanted to have the dashboard exposed on my default https entrypoint, with a custom path, as explained at "https://docs.traefik.io/configuration/api/", si I created a new .toml file as following:
```toml
[backends]
  [backends.traefik]
    [backends.traefik.servers.traefik]
      url = "http://localhost:8080"

[frontends]
  [frontends.traefikAPI]
    backend = "traefik"
    [frontends.traefikAPI.routes.traefikAPI]
      rule = "PathPrefixStrip:/traefik"
```
### What did you expect to see?

With the previous WebUI, I had no problems (up to 1.6.0-RC6), and I did not expect any by upgrading to 1.6.0.
I could see my dashboard, with the proviedres at https://host.mydomain.com/traefik/dashboard/

### What did you see instead?

With 1.6.0, https://host.mydomain.com/traefik/dashboard/ shows "No providers found.", and spawns http 404 errors because it requests '/api/providers' which is routed to another project.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.6.0
Codename:     tetedemoine
Go version:   go1.10.1
Built:        2018-04-30_09:28:44PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

<details>
<summary>config</summary>

```json
{
 "LifeCycle": {
  "RequestAcceptGraceTimeout": 0,
  "GraceTimeOut": 0
 },
 "GraceTimeOut": 0,
 "Debug": false,
 "CheckNewVersion": true,
 "SendAnonymousUsage": false,
 "AccessLogsFile": "/var/log/traefik/access.log",
 "AccessLog": null,
 "TraefikLogsFile": "/var/log/traefik/traefik.log",
 "TraefikLog": null,
 "Tracing": null,
 "LogLevel": "INFO",
 "EntryPoints": {
  "http": {
   "Address": "xxxx",
   "TLS": null,
   "Redirect": {
    "entryPoint": "xxxx"
   },
   "Auth": null,
   "WhitelistSourceRange": null,
   "WhiteList": null,
   "Compress": false,
   "ProxyProtocol": null,
   "ForwardedHeaders": null
  },
  "https": {
   "Address": "xxxx",
   "TLS": {
    "MinVersion": "",
    "CipherSuites": null,
    "Certificates": [],
    "ClientCAFiles": null,
    "ClientCA": {
     "Files": null,
     "Optional": false
    }
   },
   "Redirect": null,
   "Auth": null,
   "WhitelistSourceRange": null,
   "WhiteList": null,
   "Compress": false,
   "ProxyProtocol": null,
   "ForwardedHeaders": null
  },
  "traefik": {
   "Address": "xxxx",
   "TLS": null,
   "Redirect": null,
   "Auth": {
    "Basic": {
     "Users": null,
     "UsersFile": "xxxx"
    },
    "Digest": null,
    "Forward": null,
    "HeaderField": ""
   },
   "WhitelistSourceRange": null,
   "WhiteList": null,
   "Compress": false,
   "ProxyProtocol": null,
   "ForwardedHeaders": null
  }
 },
 "Cluster": null,
 "Constraints": [],
 "ACME": null,
 "DefaultEntryPoints": [
  "http",
  "https"
 ],
 "ProvidersThrottleDuration": 2000000000,
 "MaxIdleConnsPerHost": 200,
 "IdleTimeout": 0,
 "InsecureSkipVerify": true,
 "RootCAs": null,
 "Retry": null,
 "HealthCheck": {
  "Interval": 30000000000
 },
 "RespondingTimeouts": null,
 "ForwardingTimeouts": null,
 "AllowMinWeightZero": false,
 "Web": null,
 "Docker": null,
 "File": {
  "Watch": true,
  "Filename": "",
  "Constraints": null,
  "Trace": false,
  "TemplateVersion": 0,
  "DebugLogGeneratedTemplate": false,
  "Directory": "/etc/traefik/rules.d/"
 },
 "Marathon": null,
 "Consul": null,
 "ConsulCatalog": null,
 "Etcd": null,
 "Zookeeper": null,
 "Boltdb": null,
 "Kubernetes": null,
 "Mesos": null,
 "Eureka": null,
 "ECS": null,
 "Rancher": null,
 "DynamoDB": null,
 "ServiceFabric": null,
 "Rest": null,
 "API": {
  "EntryPoint": "traefik",
  "Dashboard": true,
  "Debug": true,
  "CurrentConfigurations": null,
  "Statistics": null
 },
 "Metrics": null,
 "Ping": null,
 "ConfigFile": "/etc/traefik/traefik.toml"
}
```

</details>

### If applicable, please paste the log output at DEBUG level (`--logLevel=DEBUG` switch)

```
(paste your output here)
```

**Comment by rroblik** [comment]
Yeah. If anybody want to do the same directly with docker `labels` : https://github.com/containous/traefik/pull/3299#issuecomment-388288678

**Comment by traefiker** [comment]
Closed by #3299.
